.github/workflows/deploy.yml

```
@@ -40,13 +44,15 @@ jobs:
           chmod +x $HOME/.local/bin/ecs-deploy
           
       - name: Login to GitHub Container Registry
+        if: ${{ inputs.PUSH_TO_ECR == true }}
         uses: docker/login-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)